### ISSUE TITLE
Add action_trace block_num index to mongo_db_plugin

### ISSUE THREAD

**Issue by heifner** [EOSIO/eos#6484]
- Add block_num as one of the default indexes for action_traces in mongo_db_plugin.
- Remove trx_id action_trace index

**Comment by yaanhyy** [comment]
Hi,  heifner. Now we meet a troublesome problem. The eos actions contained by ont block is larger and larger. Our node cannot sync data to mongodb to keep pace with the block produced. Can I use below config to speed up： 
```
mongodb-store-action-traces = false
mongodb-store-blocks = false
mongodb-store-block-states = false
```
what about read_only?
Thanks for help!

**Comment by heifner** [comment]
You should be using read_only on the nodeos with the mongo_db_plugin.
Adding those options will speed it up, yes.

**Comment by heifner** [comment]
Resolved by #6499
